lines.js

```diff
@@ -8,21 +8,37 @@ var sizePanel = document.querySelector("#dot-size-panel");
 var dotSize;
 var radius = document.querySelector(".dot-size.chosen").dataset.size;
 var mousePressed = false;
+var projectHistory = [];
+var historyBack = document.querySelector("#history-backward");
+var historyForward = document.querySelector("#history-forward");
+var historyChanges = 0;
+var currentIndex = historyChanges;
+var isOut;
```

Comment:
I can not realize intent of the variable. Choosing point or line? Then name is strange.

Reply:
Which variable?

Reply:
`isOut`. Out from where?

Reply:
Out from canvas-element

Reply:
Okay, let's name it so